tools/run_tests/performance/remote_host_prepare.sh

```diff
@@ -41,10 +41,9 @@ ssh "${USER_AT_HOST}" "rm -rf ~/performance_workspace && mkdir -p ~/performance_
 # could also kill jenkins.
 ssh "${USER_AT_HOST}" "killall -9 qps_worker mono node ruby || true"
```

Comment:
technically, no. but let's leave it to minimize the chance we don't get garbage data out of the multi-machine setup.